desc/compute/_profiles.py

```diff
@@ -1474,3 +1474,33 @@ def _current_r(params, transforms, profiles, data, **kwargs):
 def _current_rr(params, transforms, profiles, data, **kwargs):
     data["current_rr"] = 2 * jnp.pi / mu_0 * data["I_rr"]
     return data
+
+
+@register_compute_fun(
+    name="shear",
+    label="\\rho \\partial_{\\rho} \\iota / \\iota",
+    units="~",
+    units_long="None",
+    description="Normalized derivative of the rotational transform",
+    dim=1,
+    params=[],
+    transforms={"grid": []},
+    profiles=[],
+    coordinates="r",
+    data=["0", "iota", "iota_r", "iota_rr", "rho"],
+    axis_limit_data=["iota_den_r", "iota_num_r"],
+)
+def _shear(params, transforms, profiles, data, **kwargs):
+    eps = 1e2 * jnp.finfo(jnp.array([1.0]).dtype).eps
+    data["shear"] = cond(
+        jnp.all(jnp.abs(data["iota"])) < eps,
+        lambda _: data["0"],  # if iota profile is all 0, set shear to 0
+        lambda _: jnp.where(  # apply l'Hopital rule where necessary
+            (jnp.abs(data["iota"]) < eps)
+            & ((jnp.abs(data["rho"]) < eps) | (jnp.abs(data["iota_r"]) < eps)),
+            1 + data["rho"] * data["iota_rr"] / data["iota_r"],  # limit
+            data["rho"] * data["iota_r"] / data["iota"],  # shear
+        ),
+        1,
```

Comment:
We've been using `None` as a dummy argument in other places, I'd prefer that to `1` which might be confused for an actual value.